HOW TO USE:
Delete settings to take the signature photo, and restore original settings
$ patch -R < signaturePhoto.patch
Make settings take the signature photo
$ patch < signaturePhoto.patch

diff --git a/Ding-Lab - m5231142/drlFX/papers/latex/m5231142.tex b/Ding-Lab - m5231142/drlFX/papers/latex/m5231142.tex
index 5c6fc26f..f1df2cf6 100644
--- a/Ding-Lab - m5231142/drlFX/papers/latex/m5231142.tex	
+++ b/Ding-Lab - m5231142/drlFX/papers/latex/m5231142.tex	
@@ -40,6 +40,7 @@
 \usepackage{listings}  % For inserting programming code
 \usepackage{docmute}   % For compilation of divided files
 \usepackage{CJKutf8}   % Japanese
+\usepackage{tikz}      % For the approved signature page
 
 % % % Floating figure and table
 \makeatletter
diff --git a/Ding-Lab - m5231142/drlFX/papers/latex/master.sty b/Ding-Lab - m5231142/drlFX/papers/latex/master.sty
index bf41968f..9a337d5a 100644
--- a/Ding-Lab - m5231142/drlFX/papers/latex/master.sty	
+++ b/Ding-Lab - m5231142/drlFX/papers/latex/master.sty	
@@ -332,79 +332,11 @@
 %
 % % %
 \def\makeapproval{%
-  \ThisCenterWallPaper{0.45}{./Figure/emblem2g}
-  \setlength{\baselineskip}{14pt}     
   \thispagestyle{empty}
-  \vspace*{0.8cm}
-
-  \begin{center}
-    The thesis titled%\par
-  \end{center}
-
-  \vspace{0.4cm}
-
-  \begin{center}
-    \@ifundefined{@atitle}{%
-      {\large \setlength{\baselineskip}{14pt}\textsl{\@title}}%\par
-    }{%
-      {\large \setlength{\baselineskip}{14pt}\textsl{\@atitle}}
-    }%
-  \end{center}
-
-  \vspace{0.2cm}
-
-  \begin{center}
-    by\par
-  \end{center}
-
-  \begin{center}
-    {\large \@author}% \par
-  \end{center}
-
-  \begin{center}
-    is reviewed and approved by:%\par
-  \end{center}
-
-  \vspace{1.0cm}
-  \begin{table}[!h]
-    \tabcolsep=1pt
-    %{\large
-    \renewcommand{\arraystretch}{2.0}
-    \centering
-    \begin{tabular}{l} 
-      \hspace{14.0cm} \\ 
-      \hline
-      \textbf{Main referee} \\ 
-      \textit{\@mainrefereeposition} \\             
-      \hspace*{1em}\textsl{\@mainreferee} \\
-      \hline 
-      \textit{\@secondrefereeposition} \\
-      \hspace*{1em}\textsl{\@secondreferee} \\
-      \hline
-      \textit{\@thirdrefereeposition} \\
-      \hspace*{1em}\textsl{\@thirdreferee} \\
-      \hline
-    \end{tabular}
-    %}
-  \end{table}
-
-  \vspace{0.1cm}
-
-  \begin{figure}[!h]
-    \centering
-    \includegraphics[height=3mm]{./Figure/mono3g}%The University of Aizu\par
-  \end{figure}
-
-  \vspace{0.1cm}
-
-  \begin{center}
-    \textsl{\@datestamp}\par
-  \end{center}
-  \ClearWallPaper
-  \newpage
-
-  \global\let\@atitle\@empty
-  \global\let\atitle\relax
+  \begin{tikzpicture}[remember picture,overlay]
+    \node at (current page.center) {\includegraphics[width=\pdfpagewidth,height=\pdfpageheight]{./Figure/signature.jpg}};
+  \end{tikzpicture}
+  \clearpage
 }
 %
 
